Whats_New/whats-new.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<whatsnew>
```

Comment:
It is better that the url does not have region information.
It will be auto added when browsing.
e.g. https://www.renesas.com/document/rln/smart-configurator-rh850-v140-release-note

Reply:
I've removed "tw" and "en" from the hyperlinks.